geoalchemy2/elements.py

```diff
@@ -219,6 +219,12 @@ def _data_from_desc(desc):
             desc = desc.encode(encoding="utf-8")
         return binascii.unhexlify(desc)
 
+    def __eq__(self, other):
+        return self.desc == other.desc
```

Comment:
Shouldn't we also compare the `extended` and `srid` values here?
```suggestion
        return self.desc == other.desc and self.srid == other.srid and self.extended == other.extended
```